wrpcorrectify/correctify.go

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2022 Comcast Cable Communications Management, LLC
+// SPDX-License-Identifier: Apache-2.0
+
+package wrpcorrectify
+
+import (
+	"context"
+
+	"github.com/xmidt-org/wrp-go/v3"
+)
+
+// Correctifier applies a series of normalizing options to a WRP message.
+type Correctifier struct {
```

Comment:
This type stutters:  `wrpcorrectify.Correctifier`.  The usual way to do this in `golang` is to call this type `T`: `wrpcorrectify.T`.  The idea is that it's a central type to the package.  I'm dubious about this, but it is what's done.

Probably a better way is to rename either the type or the package to something that doesn't stutter.

Reply:
Yeah, these were just working names.